### PR TITLE
random GUID assigned during load

### PR DESCRIPTION
**Pull request by MoltonMontro**
Added a note that the game will automatically try to assign a random GUID during load, if the asset is loaded successfully.

### CODE REVIEW

**Comment by SDGNelson** [comment]
Thanks, documenting this is a good idea. I think the wording may be a little confusing however. The original purpose was to add guids to all preexisting asset files. Maybe something along the lines of "if the GUID property is omitted"?

**Comment by MoltonMontro** [comment]
Does this generation apply to _all_ asset files? Or is it more narrow, such as just for AssetV1 files?

**Comment by SDGNelson** [comment]
Yes, only v1 at the moment unfortunately

**Comment by MoltonMontro** [comment]
Changes made based on clarification. Pull request is commit-ready.

**Comment by SDGNelson** [comment]
Thanks!

File: GUID.md [hunk 1]
```diff
@@ -3,6 +3,6 @@ GUID
 
 Globally Unique Identifiers (**GUID**s) are 32 hexadecimal digits used to identify assets. They are preferable to file names because the files can be moved without redirectors.
 
-At the time of writing a useful tool for generating GUIDs is: https://www.guidgenerator.com/
+A useful tool for manually generating GUIDs is [guidgenerator.com](https://www.guidgenerator.com/). Note that if the GUID property is omitted from an [asset v1](/AssetV1.md) file, then the game will automatically assign a random GUID during a successful load.
```
Comment (CyberAndrii) [inline GUID.md:6]: Should be `AssetsV1.md` instead of `AssetV1.md` (

**Pull request merged**